packages/components/src/styles/components/app-header.scss

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// APP-HEADER
+//
+
+@use "../mixins/interactive-dark-theme" as *;
+
+// ----------------
+// Q: Create tokens?
+$app-header-home-link-padding: 4px;
+
+// Breakpoints WIP: START ----------------------------
+
+// Break points
+$tablet-min-width: 768px;
```

Comment:
Is this breakpoint being used? Or is this a leftover from some of the earlier design explorations.

Reply:
Left over. Cleaned everything up in the latest push.